8-show-manifest-and-values/README.md

```diff
@@ -0,0 +1,333 @@
+
+### Goals
+
+<!--
+List the specific goals of the ZEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+This is successful for creators when they can view manifests or values files before building their package to gain confidence that their variables are templated as intended before creating their packages.
+
+This is successful for deployers when they can view their templated manifests and values files before deploying.
+
+### Non-Goals
+
+<!--
+What is out of scope for this ZEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+Zarf will not accept a package in the cluster as input to zarf package show-manifests or zarf package show-values-files. While certain Zarf commands, such as `zarf package inspect`, do allow the cluster as a package source, accepting cluster sources offers limited value. This is because variable templating is already complete, and users can view their deployed charts using [helm get manifest](https://helm.sh/docs/helm/helm_get_manifest/)
+
+## Proposal
+
+<!--
+This is where you explain the specifics of the proposal. Provide enough detail
+for reviewers to clearly understand what you're proposing, but avoid including
+too many specifics like API designs or implementation details. Focus on the
+desired outcome and how success will be measured. The "Design Details" section
+below is for the real nitty-gritty.
+-->
+
+Introduce four new commands. `zarf dev show-manifests`, `zarf dev show-values-files`, `zarf package show-manifests`, and `zarf package show-values-files`. The `package` commands will run on an already built package, local or remote, while the `dev` commands will take a package directory. This command will print both the manifests from a helm chart and the manifests in the `.components[x].manifests` key. Component actions will not run during any of these commands.
+
+Before printing the manifest for each chart the name and version of the chart will be printed. Before printing Manifests from the `.components[x].manifests` key the name of the manifests block, `.components[x].manifests[x].name`, will be printed.
+
+Below is the intended help text for `zarf dev show-manifests`. `zarf dev show-values-files` will include the same flags.
+```
+Usage:
+  zarf dev show-manifests [ DIRECTORY ] [flags]
+
+Flags:
+      --create-set stringToString   Specify package variables to set on the command line (KEY=value) (default [])
+      --deploy-set stringToString   Specify deployment variables to set on the command line (KEY=value) (default [])
+  -f, --flavor string               The flavor of components to include in the resulting package (i.e. have a matching or empty "only.flavor" key)
+      --kube-version                Override the default helm template KubeVersion when performing a package chart template
+      --confirm                     Confirms command without prompting. Skips prompts to configure variables.
```

Comment:
Again, `--confirm` in a preview seems like an overkill.

Reply:
I think it's reasonable given that it would only prompt if there are variables that have the `prompt: true` key or templates that are not set (we error when templates aren't set)